scripts/EN/Harry

```diff
@@ -5710,48 +5710,48 @@ CleanScreen(); // 0x7348: 00000064
 LoadBackground(0x0); // 0x734c: 00001069
 DrawBackground(); // 0x7352: 00000065
 OP_0x29(); // 0x7356: 00000029
-ShowText((1602)"¥"Gu...¥" Cybil tries to speak, and her face twists agonizingly. At the same time white smoke rises from her entire body, and gravity pulls her to the ground, face down."); // 0x735a: 0000100c
+ShowText((1602)"¥"Uggh…¥"¥nW2Cybil tries to speak, W1and her face twists agonizingly.¥nW2At the same time white smoke rises from her entire body, W1and gravity pulls her to the ground, face down."); // 0x735a: 0000100c
```

Comment:
Changed GU to English "Uggh"

Feel free to insert the onomatopoeia for pain of your choice here.

Uggh is the second thing Cybil says in the PS1 game, so we could change to "Uh-ah".

If your language has something short that includes the sound goo that would be best I think.